### DISCUSSION TITLE
δεν φαινεται η ανακοινωση μου

### DISCUSSION THREAD

**Discussion by p16andr** [ioniodi/sitegr#166] [Q&A] [answered]
@epidrome  @c15pati 
Για το το αιτημα ενσωματωσης οσον αφορα το Netlify πρεπει να κανω μονον deploy from your branch το md αρχειο μου.
Το κανω και για καποιο λογο η ανακοινωση μου δεν φαινεται. Μπορειται να με βοηθησετε λιγο σας παρακαλω;;

**Comment by epidrome** [comment]
έχεις διαβάσει τις οδηγίες;
https://github.com/ioniodi/sitegr/wiki/Workflow

από όσα γράφεις δεν καταλαβαίνω το πρόβλημα

μετέφερα το θέμα σε συζήτηση εδώ μήπως κάποιος άλλος το έχει αντιμετωπίσει και μπορεί να βοηθήσει

πάντως οι περισσότεροι δεν φαίνεται να έχουν πρόβλημα, για ψάξτο λίγο ακόμη και γράψε μας πάλι

**Reply by p16andr** [reply]
@epidrome 
Ναι κυριε Χωριανοπουλε εχω ακολουθησει ολη την διαδικασια με τις οδηγιες που στειλατε εδω και ακολουθησα μετα και την διαδικασια με την δημιουργια νεου branch απο το fork μου στο sitegr, προσθεσα στα  all_collections/_posts την ανακοίνωση μου  https://github.com/p16andr/sitegr/tree/2016063/all_collections/_posts  και εκανα deply from my branch την ανακοινωση μου στο Netlify και οταν παταω preview για να την δω δεν φαινεται η ανακοινωση μου.

![2021-03-20 (2)](https://user-images.githubusercontent.com/72922731/111867190-1e768a80-897b-11eb-9791-fad0e1db126b.png)

**Reply by epidrome** [reply]
θα έπρεπε να φαίνεται κάτι άλλο φταίει και μας διαφεύγει

δοκίμασε να κάνεις κάποια άλλη αλλαγή σε κάποιο άλλο δοκιμαστικό κλαδί και γενικά ψάξτο με δοκιμές

**Reply by p16andr** [reply]
@epidrome 
 Κυριε Χωριανοπουλε εκανα και αλλαγη σε αλλο branch και γενικα καποιες μικροαλλαγες και εκανα πολλες δοκιμες και παλι δεν μου εμφανιζει την ανακοινωση μου ενω κανω deply. Δεν καταλαβαινω τι μπορει να μας διαφευγει.

**Reply by epidrome** [reply]
υπάρχουν άπειρα πράγματα που μπορεί να πάνε στραβά συνήθως είναι κάτι απλό και προφανές

γιατί δεν ρωτάς κάποιον από τους συναδέλφους σου σε ένα από τα πολλά φροντιστήρια της πέμπτης?

αν δεν κάνω λάθος δεν μπήκες σε κανένα από τα πρόσφατα...

**Comment by p16andr** [comment, marked as answer]
Νταξει λυθηκε το θεμα που ειχα με την μη εμφανιση της ανακοινωσης στην demo ιστοσελιδα μου με τις απαραιτητες ενεργειες που μου ανεφερε ο συμφοιτητης μου @NikosKar23. 
1) Στο excerpt εκλείσα τα εισαγωγικά που ειχα ανοίξει (ακριβώς πριν τα tags, πρέπει να κλείνουν)
2) Λίγο πριν το τέλος, εκεί που έχει τις 3 παύλες (---) υπυρχε ένα κενό και εκάνα ένα backspace  και  
3)  Στον τίτλο του αρχείου αντί για 09-02-2021, χρησιμοποιήσα την μορφολογία που υπάρχει στο site, δηλαδή πρώτα η χρονιά μετά ο μήνας και μετά η ημέρα. (Δηλ: 2021-02-09)
